style.css

```diff
@@ -93,3 +93,72 @@ hr {
 #author::placeholder {
   color: black;
 }
+
```

Comment:
- [ ] Your application is awesome. But I think there is a bit of a difference between your design and the wireframe. For example your element `width` and also I noticed that the time and date after your navbar are missing. Please fix it in order to make your work completed. :smile: